sdk/data/azcosmos/cosmos_container.go

```diff
@@ -358,6 +358,50 @@ func (c *ContainerClient) ReadItem(
 	return newItemResponse(azResponse)
 }
 
+// PatchItem patches an item in a Cosmos container.
+// ctx - The context for the request.
+// partitionKey - The partition key for the item.
+// itemId - The id of the item to patch.
+// item - The content to be used to patch.
```

Comment:
[From API review]  These will be operations to probably the text would be: 

```suggestion
// ops - Operations to perform on the patch
```